### ISSUE TITLE
Explain how 'trending' is determined

### ISSUE THREAD

**Issue by keunes** [Podcastindex-org/docs-api#67]
Hello,

I proposed to replace the iTunes podcast suggestions in AntennaPod with suggestions based on the Podcast Index. One of the arguments was PI cannot provide a list of podcasts popular in the user's country. However, there is a 'Trending' end-point now, so I assume we could use that.

How is defined, though? What makes a podcast Trending? (It'd be great if that could be described also in the docs.)

>  Trending
> get /podcasts/trending
> This call returns the podcasts/feeds that in the index that are trending.

Thanks!

**Comment by daveajones** [comment]
> Nice! What do they provide? A ranking based on the number of subscribers? Do they report actual numbers; is it real time? Is it 'global' or country-level data?

I have those same questions!  They are still building out their systems so we don’t even know yet.  All we would really need is subscriber counts. Something like <feedurl>=n would be just fine.

**Comment by keunes** [comment]
As a follow-up on this, here's some points of improvement for this end-point: https://github.com/AntennaPod/AntennaPod/issues/5986

**Comment by josh-ashkinaze** [comment]
Hi, is it still the case that the trending rank comes purely from how much traffic the api sees for a podcast? I am wondering for a research project. 

P.S: Even if it's hard to get other data, I can see using the change in API calls at least as a more nuanced measure of trending.

**Comment by daveajones** [comment]
Yes, this is correct.  It's a curated list of apps that influence the ranking.  This is also the "popularity" score in our weekly database download.
